bottlecap/src/lifecycle/invocation/processor.rs

```diff
@@ -174,6 +185,13 @@ impl Processor {
                 proactive_initialization.to_string(),
             );
         }
+
+        if let Some(runtime) = &self.runtime {
+            self.span
+                .meta
+                .insert(String::from("runtime"), runtime.to_string());
+        }
```

Comment:
Not sure if we actually need this, it looks like previous versions have the `runtime` tag in spans, but not sure if this is done in backend or tracer.

Reply:
We do need it